lib/tty/pager.rb

```diff
@@ -63,6 +65,15 @@ def enabled?
       !!@enabled
     end
 
+    # Check if pager is currently running
+    #
+    # @return [Boolean]
+    #
+    # @api public
+    def running?
```

Comment:
In what scenario would this be useful?

Reply:
I created it to have a single way to check if the pager process was started, so that both `write` and `wait` could respond to the state of the pager. Right now, it's not actually used, because I check for the presence of the `PagerIO`, so I guess I'll remove it.